components/Apoiador/index.jsx

```
@@ -1,194 +1,108 @@
 
-  function validatePhone(phone) {
-    const phoneRegex = /^\(*\d{2}\)*(\s)?\d{4,5}-*\d{4}$/;
-    return phoneRegex.test(phone);
-  }
-
-  function validateConfirmPhone(confirmPhone) {
-    return confirmPhone === phone;
-  }
+  const onSubmit = async (values, { resetForm }) => {
+    {!dataAccepted && setShowDataModal(true)}
+    
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)